module-2/lab-df-calculation-and-transformation/your-code/challenge-1.ipynb

```diff
@@ -394,34 +394,6 @@
    "cell_type": "code",
```

Comment:
- Print the distinct values in Type 1 and Type 2 combined: Concat function does not throw the result we want here. The key is "combined", take a look on the following: 
```Python
distinct_values=pokemon.groupby(["Type 1", "Type 2"]).size().reset_index().rename(columns={0:'count'})
distinct_values.head(10)
```
- Another way of combining Type1 and Type2 without using def or lambda functions. Numpy has several useful methods, I would suggest to explore them when you have some free time.
```Python
pokemon['Combo Type']= np.where(pokemon['Type 2'].isnull(),pokemon['Type 1'],pokemon['Type 1'] +'-'+pokemon['Type 2'])
```
- Another way of getting the guessed total column: 
```Python
pokemon['Guessed Total'] = pokemon.loc[:,'HP':'Speed'].sum(axis = 1)
```